Gemfile.lock

```diff
@@ -321,5 +321,8 @@ DEPENDENCIES
   webmock (~> 3.6.0)
   yard
 
+RUBY VERSION
+   ruby 2.6.3p62
```

Comment:
Should this different to the `2.6.3` in `.ruby-version` file?

Reply:
It should be `2.6.3`, but patches are allowed it seems.

Reply:
Interesting, thanks!